dali/CMakeLists.txt

```diff
@@ -122,6 +125,7 @@ if (BUILD_TEST)
   add_dependencies(${test_main_bin} ${dali_lib})
 
   # We'll have to add dependency libs
+  target_link_libraries(${test_main_bin} PRIVATE "-Wl,--whole-archive $<TARGET_LINKER_FILE:${dali_kernel_test_lib}> -Wl,--no-whole-archive")
```

Comment:
I don't know if we care about this, it is executable so it won't expose symbols anyway.

Reply:
It's crucial, since GTEST tests are instantiated as global variable side effects...

Reply:
BTW - it's not working on CMake level, fix is pending...